docs/architecture.md

```diff
@@ -102,33 +102,10 @@ Runtime supports the following auths
 
 To know how to add a new auth, see this doc {@tutorial new-auth-mechanisms}.
 
-Each auth has four hooks
+Each auth implements the 4 hooks in {@link AuthHandlerInterface}
 - pre
 - init
 - sign
 - post
 
-#### pre
-This hook decides whether all the required paramaters are present in the auth
```

Comment:
Why removed?

Reply:
coz the information is now in the `AuthHandlerInterface` 
added a link here instead